mappings/cwe.json

```diff
@@ -311,11 +311,11 @@
     },
     {
       "id": "external_behavior",
-      "cwe": ["CWE-2000"]
+      "cwe": null
```

Comment:
what is the thought on making this null instead of just removing the entry?

Reply:
It won't pass the tests as an entry with no mapping

Reply:
Ah interesting, because it is a category level it is required.  This seems like a strange requirement especially when we have a default, but I suppose it does prevent mistakes.

In the vrt-ruby gem it would happily accept a mapping with a missing category level and return the default.